eth_name: eth0
slave_no: [1, 2, 3]
esi_file: elfin_module.esi
